packages/cozy-harvest-lib/src/helpers/accounts.js

```diff
@@ -94,6 +96,55 @@ export const mergeAuth = (account, authData) => ({
   auth: merge(account.auth, authData)
 })
 
+/**
+ * Gets the vault cipher relationship to an account
+ * @param  {object} account   io.cozy.accounts document
+ * @return {string}           cipher uuid
+ */
+export const getVaultCipherId = account =>
+  account.relationships &&
```

Comment:
lodash.get is pretty good for those cases: `get(account, 'relationships.vaultCipher._id')`